pkgs/by-name/te/tetragon/package.nix

```diff
@@ -7,68 +7,78 @@
   llvm_16,
   clang_16,
   bash,
+  writableTmpDirAsHomeHook,
+  gitMinimal,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "tetragon";
-  version = "1.2.0";
+  version = "1.4.0";
 
   src = fetchFromGitHub {
     owner = "cilium";
     repo = "tetragon";
     tag = "v${finalAttrs.version}";
-    hash = "sha256-KOR5MMRnhrlcMPqRjzjSJXvitiZQ8/tlxEnBiQG2x/Q=";
+    hash = "sha256-HOVQdKUIBLq9/2hTokZKvLZOgRQu5/lAwYy1yQa1bus=";
   };
 
+  nativeBuildInputs = [
+    writableTmpDirAsHomeHook
```

Comment:
This should only be added in `nativeCheckInputs`. There's no need to put it in `nativeBuildInputs`.

Reply:
Hooo interesting, fair enough, leave it there